packages/core/src/index.ts

```diff
@@ -39,6 +40,7 @@ const API = {
       setLocale,
       updateNotify,
       customNotification,
+      preflightNotification,
```

Comment:
I wonder if the function should be called `preflightNotifications` instead to indicate that it renders a series of notifications rather than just one?

Reply:
Good point! Ill update